openedx/features/course_experience/views/course_home.py

```diff
@@ -105,6 +105,7 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         context = {
             'csrf': csrf(request)['csrf_token'],
             'course_key': course_key,
+            'course': course,
```

Comment:
@andy-armstrong Is the fix for this issue being handled in another one of your PRs?